tool/services/github_service.py

```diff
@@ -118,4 +118,34 @@ def create_pull_request(self, target_remote, target_branch, current_branch):
         return pr.html_url
 
     def list_pull_requests(self):
+        """
+        Get list of pull-requests for the repository.
+
+        :return: [PullRequest]
+        """
         return list(self.repo.get_pulls(state="open", sort="updated", direction="desc"))
+
+    def list_labels(self):
+        """
+        Get list of labels in the repository.
+        :return: [Label]
+        """
+        return list(self.repo.get_labels())
+
+    def update_labels(self, labels):
```

Comment:
what do we do with labels which are in current repo and not in `labels`?

Reply:
That's a question. Some `--force` option?

Some labels can differ only a little (`good first issue` x `good-first-issue`) and contain some issues already...

Reply:
okay, that makes sense, let's not implement the purge functionality yet then